src/response.rs

```diff
@@ -64,7 +122,58 @@ impl Response {
         }
     }
 
-    /// Get the `Headers`.
+    /// Get the `Headers` of this `Response`.
+    ///
+    /// ```rust
+    /// # extern crate reqwest;
+    /// # #[macro_use] extern crate error_chain;
+    /// # use std::fs;
+    /// # use std::io::{Read, Write};
+    /// # use reqwest::{Client, StatusCode};
+    /// # use reqwest::header::ContentType;
+    /// # use reqwest::mime::{Mime, TopLevel, SubLevel};
+    /// # error_chain! {
+    /// #     foreign_links {
+    /// #         Reqwest(reqwest::Error);
+    /// #         Io(::std::io::Error);
+    /// #     }
+    /// # }
+    /// # fn run() -> Result<()> {
+    /// let client = Client::new()?;
+    /// let mut resp = client.get("https://img.shields.io/crates/v/mime.png?label=mime")?.send()?;
```

Comment:
I think a simpler example like checking `ContentLength` would be best. The clutter of matching against the mimetype is definitely a useful example, but might be better in a standalone example.